### ISSUE TITLE
Authentication error - Unexpected character

### ISSUE THREAD

**Issue by harshuchilzy** [RayOkaah/woocommerce#13]
There's an error "Unexpected character" throwing then I try to use 
`final customer = woocommerce.loginCustomer(username: username, password: password);`

**Comment by GitGud31** [comment]
Yup, I'm having the same issue aswell. Please let me know if you find a solution
